src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -897,8 +897,10 @@ def internal_evaluate_expression_json(py_db, request, thread_id):
         elif context == 'repl':
             try:
                 pydevd_vars.evaluate_expression(py_db, frame, expression, is_exec=True)
-            except:
-                pass
+            except Exception as ex:
```

Comment:
It'd be nice to have a test to cover for this use-case.

Reply:
Will add.